pkg/controller/migration/convert/network.go

```diff
@@ -65,7 +81,7 @@ func handleNetwork(c *components, install *Installation) error {
 	if am != nil {
 		tam, err := getAutoDetection(*am)
 		if err != nil {
-			return ErrIncompatibleCluster{"error parsing IP_AUTODETECTION_METHOD: " + err.Error()}
+			return ErrIncompatibleCluster{fmt.Sprintf("%s, error parsing IP_AUTODETECTION_METHOD: "+err.Error(), context)}
```

Comment:
nit, you're already using fmt.Sprintf - why not go the extra mile?
```suggestion
			return ErrIncompatibleCluster{fmt.Sprintf("%s, error parsing IP_AUTODETECTION_METHOD: %s", context, err)}
```